service/routers/decks.js

```diff
@@ -58,11 +66,17 @@ const deleteDeck = async (req, res) => {
   const userId = ''
   const deckId = req.params.id
   try {
-    const user = await User.findById(userId)
-    const removedDeck = user.decks.id(deckId).remove()
-    console.log(removedDeck)
-    user.save()
-    res.sendStatus(204)
+    const user = await User.findById(req.params.id)
```

Comment:
It seems like you're trying to find a user by a deck's id here. You can use mongoose's find operation to find a user with a subdocument matching an id. You should be able to use `{ "decks._id": req.params.id }` as a query, though the question then becomes checking the id of the returned user and the auth user to make sure they should be able to interact.